composer.json

```
@@ -31,7 +31,7 @@
 		"ext-libxml": "*",
 		"ext-mbstring": "*",
 		"ext-simplexml": "*",
-		"league/flysystem": "^1.0",
+		"league/flysystem": ">=1.1.4 <2.0.0",
```

A more clean and concise equivalent would be the following.

```suggestion
		"league/flysystem": "^1.1.4",
```